app/Models/Comment.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Comment extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'content',
+        'user_id',
+        'kebab_place_id',
+        'parent_id'
+    ];
+
+    public function user()
```

Comment:
```suggestion
    public function user(): BelongsTo
```
along with import BelongsTo

please, check all functions again and add return types.